Various aesthetic updates/changes for Slimes

Salem-uwu opened the pull request:
Adds a LOT of aesthetic updates for the Slimes (Now Laspii) AND some generic markings for other species now

## Why we need to add this
Slimes are so cosmetically underwhelming, and more customization is ALWAYS a good thing!! Also slime people are a dumb name and feel out of place in the world :3

## Media (Video/Screenshots)
https://github.com/user-attachments/assets/1b2adad3-f611-49b5-848e-b72f3bbd720e

## Checks
- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: Salem and Arkanic
- add: Eyes as a marking slot
- add: Added Impstation Slime Markings, and base sprite with permission from all those involved
- add: Added two new hair styles that work well with Slimes, also from imp and ported with permission
- add: Ported a bunch of generic markings by sha-seng (human, dwarf, lizard, slime, vulp)
- tweak: Renamed Slime People to Laspi (Singular), Laspii (Plural), and Laspians (Demonym) (Only on the UI, so this shouldn't cause any file-related problems)

Arkanic committed:
slime markings

Commit:
Various slime changes, and 2 new hair styles

Salem-uwu commented:
ALSO, I did get permission to add the sprites from all those involved!!! :3

![image](https://github.com/user-attachments/assets/11405ed5-95fd-42eb-a1c8-afd013bd3476)

![image](https://github.com/user-attachments/assets/957336de-98f5-417f-ab0c-15543aff0e3d)

![image](https://github.com/user-attachments/assets/28acdb1c-352e-4cda-986a-6f65c0849571)

Conflee commented:
Sprites are approved. YAMLINTER though is mad.

Arkanic committed:
fix

Salem-uwu committed:
Merge pull request #1 from Arkanic/human-markings

backslashes are bad

Commit:
fixed the double defined, and apparently I forgot to push a change I made way earlier fixing a locale issue with one of the added hairs

Commit:
fuck sake I forgot to stage this

Commit:
Merge branch 'Slime-Markings' of https://github.com/Salem-uwu/Starlight-Personal into Slime-Markings

Salem-uwu committed:
Merge branch 'Starlight' into Slime-Markings

Salem-uwu commented:
YAML LINTER IS APPEASED 🙏 🙏 🙏

Arkanic committed:
generic markings from sha-seng

Salem-uwu committed:
Merge pull request #2 from Arkanic/human-markings-2

additional generic markings for other species, by sha-seng

Commit:
Added the lines required (grrrr)

Commit:
Merge branch 'Slime-Markings' of https://github.com/Salem-uwu/Starlight-Personal into Slime-Markings

Arkanic commented:
Ported a bunch of generic markings by sha-seng, these can be applied (in varying numbers) to:
- humans
- dwarves
- slimes
- lizards
- vulpkanin

Includes a bunch of tattoos for all body parts, and various pieces of jewellery, have tested them all ingame

stickycheZ101 commented:
> Ported a bunch of generic markings by sha-seng, these can be applied (in varying numbers) to:
> 
> * humans
> * dwarves
> * slimes
> * lizards
> * vulpkanin
> 
> Includes a bunch of tattoos for all body parts, and various pieces of jewellery, have tested them all ingame

Does this include prosthetics?

Arkanic commented:
> Does this include prosthetics?

No, those weren't made by sha-seng (forgot who specifically, you'll have to check diff) so I didn't add them - licensing and whatnot

Salem-uwu commented:
OH FORGOT TO SAY @Conflee new sprites were added to the PR, I assume you'll need to approve them again, theyre all made by the same person though, and theyre all very high quality

Conflee commented:
new sprites also approved

Salem-uwu committed:
Forgot comments, crying

fucking forgot comment (this'll be the first one in probably multiple commits) grahhh

Salem-uwu committed:
more comments

gonna weep

Salem-uwu committed:
More comment changes again

Salem-uwu committed:
commenttsssssssssss

Salem-uwu committed:
MORE LASPII COMMENTS AARHGRHRG

Salem-uwu committed:
mhm!!! yep!!!!!

Salem-uwu committed:
how the fuck did I forget every single comment

Salem-uwu committed:
Was told to update these comments a bit ago, but forgot lmao

Salem-uwu commented:
@Happyrobot33 requesting review 🙏 (very sorry if theres a way to request review in the UI, if there is I couldnt find it)
